integration/install.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh -xe
 SUSEConnect -r $REGCODE
-SUSEConnect -p sle-module-desktop-applications/15/x86_64
-SUSEConnect -p sle-module-development-tools/15/x86_64 # this and above is needed for 'rpm-build' package
+SUSEConnect -p sle-module-desktop-applications/15.2/x86_64
+SUSEConnect -p sle-module-development-tools/15.2/x86_64 # this and above is needed for 'rpm-build' package
 zypper --non-interactive ar http://download.opensuse.org/repositories/openSUSE:/Tools/SLE_15/openSUSE:Tools.repo
 zypper --non-interactive --gpg-auto-import-keys ref
 zypper --non-interactive up
 zypper --non-interactive in -t pattern devel_osc_build
-zypper --non-interactive install --no-recommend wget curl timezone \
+zypper --non-interactive install wget curl timezone \
```

Comment:
This looks like it was a syntax error. Should the correct syntax `--no-recommends` be used instead?

Reply:
Done!